lib/RTCSession.js

```diff
@@ -617,13 +617,13 @@ module.exports = class RTCSession extends EventEmitter
     }
 
     // Don't ask for audio if the incoming offer has no audio section.
-    if (!mediaStream && !peerHasAudioLine)
+    if (!mediaStream && !peerHasAudioLine && !options.rtcOfferConstraints)
```

Comment:
We need to check *specifically* if `offerToReceiveAudio` constraint is set to not.

Also consider that `rtcOfferConstraints` is optional and it may not be present. You can create a variable as it is done for `rtcAnswerConstraints` [here](https://github.com/versatica/JsSIP/pull/624/files#diff-f0bb4b4297430ea5a3622325573c2ed9R502)